### ISSUE TITLE
Missing lforms-updater/updater.js

### ISSUE THREAD

**Issue by djvreeman** [lhncbc/lforms-fhir-app#7]
The app doesn't load any forms via upload (or either from the server). The npm console shows:

`"GET /lforms-fhir-app/lforms-updater/updater.js" Error (404): "Not found"`

and no such file is in the distribution. I think it got factored out, but perhaps is missing a dependency to pull it in?

**Comment by plynchnlm** [comment]
Hi-- Sorry for the long delay.  I just discovered that my GitHub notices were being filtered to a folder I almost never look at.  (Hopefully I have fixed that.)

I guess you are (were) probably trying to run your own version of the application, as the one we have live (https://lhncbc.github.io/lforms-fhir-app/) is working.  It sounds to me like the npm build script did not run.  If you look in package.json you will see a script that copies updater.js into place.  Did you do "npm run build"?
